feature/experimental/isis/ate_tests/base_adjacencies_test/base_adjacencies_test.go

```diff
@@ -60,23 +60,38 @@ func TestBasic(t *testing.T) {
 	}
 	isisRoot := session.ISISPath()
 	port1ISIS := isisRoot.Interface(ts.DUTPort1.Name())
-	if err := check.Equal(isisRoot.Global().Instance().State(), session.ISISName).AwaitFor(time.Second, ts.DUTClient); err != nil {
```

Comment:
What is this replaced by? Is there an alternate check that we should have that determines that IS-IS is running?

Reply:
At the global container, we configure the net and address family which is validated under the "read_config" section along with level-capability. None of the other leaf seems helpful in determining IS-IS is running.

My suggestion would be to move the "net" check here.